catalog/core/catalog-core-definitionparser/src/main/java/ddf/catalog/definition/impl/DefinitionParser.java

```diff
@@ -316,6 +354,48 @@ private void commitStaged(List<Callable<Boolean>> stagedAdds) throws Exception {
     return staged;
   }
 
+  private void processAttribute(
+      Outer.MetacardType metacardType,
+      /*Mutable*/ Set<AttributeDescriptor> attributeDescriptors,
+      /*Mutable*/ Set<String> requiredAttributes,
+      String attributeName,
+      ddf.catalog.definition.impl.DefinitionParser.Outer.MetacardAttribute attribute) {
```

Comment:
Can this just be `Outer.MetacardAttribute`?

Reply:
Done 👌 
Originally it conflicted with another MetacardAttribute class and so the auto fix filled out the full name of it 😄